src/Deserializers/Exceptions/MissingTypeException.php

```diff
@@ -12,4 +14,10 @@
  */
 class MissingTypeException extends DeserializationException {
 
+	public function __construct( $message = '', Throwable $previous = null ) {
+		if ( $message === '' ) {
+			$message = 'Type is missing';
+		}
+		parent::__construct( $message, $previous );
```

Comment:
What about

```php
public function __construct( $message = null, Throwable $previous = null ) {
    parent::__construct(
         $message ?? 'Type is missing',
         $previous
    );
```

And a parameter type for $message, or is that not possible?

Reply:
Now that I'm thinking about it, I could just do 

```php
public function __construct( $message = 'Type is missing', Throwable $previous = null ) {
    parent::__construct( $message, $previous );
}
```

After all, this is literally the default value of that parameter, might as well own that and use the syntax for it. Will add the type as well.